jsapp/js/components/bigModal/mfaModals.tsx

```diff
@@ -256,14 +264,13 @@ const MFAModals = class MFAModals extends React.Component<
               errors={this.state.errorText}
               value={this.state.inputString}
               onChange={this.onInputChange.bind(this)}
+              disableAutocomplete={true}
```

Comment:
I think linter will complain that `disableAutocomplete` is enough (i.e. without `={true}`)

Reply:
Good catch! I always forget about that 😅